persistence-jdbc/src/main/scala/zio/actors/persistence/jdbc/JDBCConfig.scala

```diff
@@ -7,14 +7,16 @@ import zio.config.ConfigDescriptor.{ nested, string }
 
 private[actors] object JDBCConfig {
 
-  final case class DbURL(value: String)  extends AnyVal
-  final case class DbUser(value: String) extends AnyVal
-  final case class DbPass(value: String) extends AnyVal
-  final case class DbConfig(dbURL: DbURL, dbUser: DbUser, dbPass: DbPass)
+  final case class DbDriver(value: String) extends AnyVal
```

Comment:
I am not a mysql user so have a silly question: Why do we need the `DbDriver` just for mysql?

Reply:
My question is: why don't we need it for postgres? :) I haven't found the answer in the code yet.
Either case the driver might be optional to preserve backward compatibility.

Reply:
@jupposessho Hi! Doobie, which we are using to connect to db, is taking care of that, so it's sufficient to specify `url` with `"jdbc:mysql://..."` for mysql so there's no need to define a driver (https://gitter.im/tpolecat/doobie?at=57dc3a26fa660dd95fe787fe)

Reply:
@mtsokol Hi! You are right, it works w/o the driver. But adding the Mysql driver is necessary in my experience.
Just for documentation purposes I created the following table:

`create table if not exists journal_zio
(
    persistence_id varchar not null,
    sequence_number unsigned int not null auto_increment,
    message bytea,
        primary key (persistence_id, sequence_number)
);`